awx/main/management/commands/create_oauth2_token.py

```diff
@@ -0,0 +1,26 @@
+from awx.api.serializers import OAuth2TokenSerializer
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from django.core.exceptions import ObjectDoesNotExist
+
+class Command(BaseCommand):
+    """Command that creates an OAuth2 token for a certain user. Returns the value of created token."""
+    help='Usage: awx-manage create_oauth2_token --user=username. Will return the value of created token.'
+
+    def add_arguments(self, parser):
+        parser.add_argument('--user', dest='user', type=str)
```

Comment:
Does `required=True` work instead of https://github.com/ansible/awx/pull/1942/files#diff-411173c5d8b94cf90375f2bc542d0088R14?

Reply:
Hey Ryan thanks for the comments. `Require=True` will throw a exception if the option is not supplied, and I do not know if this would impact our test suites for other stuff. I can change it to required=True if it does not influence other stuff.

Reply:
I agree, we should add `required=True` to the `--user` flag.  This should not impact other tests as that should only affect to this command.

Reply:
I'd still like to add this in, if it jives with what @cchurch suggested elsewhere: https://github.com/ansible/awx/pull/1942/files#r192304119